ietf-te.yang

```diff
@@ -681,21 +713,42 @@ module ietf-te {
             "Enables the hierarchical link properties supported by
              this tunnel";
         }
+        leaf local-node-id {
+          type nw:node-id;
+          description
+            "The local node identifier.";
+        }
         leaf local-te-node-id {
           type te-types:te-node-id;
           description
             "The local TE node identifier.";
         }
+        leaf local-link-tp-id {
+          type nt:tp-id;
+          description
+            "The local link termination point identifier.";
+        }
         leaf local-te-link-tp-id {
           type te-types:te-tp-id;
           description
             "The local TE link termination point identifier.";
         }
+        leaf remote-link-tp-id {
+          type nt:tp-id;
+          description
+            "The remote link termination point identifier.";
+        }
         leaf remote-te-node-id {
           type te-types:te-node-id;
           description
             "Remote TE node identifier.";
         }
+        leaf network-id {
+          type nw:network-id;
+          description
+            "The network topology identifier where the hierarchical 
+            link supported by this TE tunnel is instantiated.";
+        }
         uses te-types:te-topology-identifier {
           description
             "The topology identifier where the hierarchical link
```

Comment:
Please change the description to 'The TE topology identifier...

Reply:
Fixed https://github.com/tsaad-dev/te/pull/229/commits/4293a72724a6a311f05f7a5225cadac71dc4b542